zone/tradeskills.cpp

```diff
@@ -1866,37 +1866,59 @@ bool ZoneDatabase::DisableRecipe(uint32 recipe_id)
 bool Client::CheckTradeskillLoreConflict(int32 recipe_id)
 {
 	auto recipe_entries = TradeskillRecipeEntriesRepository::GetWhere(
-			content_db,
-			fmt::format(
-					"recipe_id = {} ORDER BY componentcount DESC",
-					recipe_id
-			)
+		content_db,
+		fmt::format(
+			"recipe_id = {} ORDER BY componentcount DESC",
+			recipe_id
+		)
 	);
 	if (recipe_entries.empty()) {
 		return false;
 	}
 
-	for (auto& f : recipe_entries) {
-		for (auto &e: recipe_entries) {
-			if (f.componentcount > 0 && e.item_id == f.item_id && e.componentcount == 0) {
-				e.item_id = 0;
+	const EQ::ItemData* f_item_inst = nullptr;
```

Comment:
Please shorten to `fi`

Still a bit verbose given this is a receiver of a complex type and makes accessing it verbose. Making it verbose doesn't exactly give more to readability

Same with below

`ei`

Reply:
done!